puppet/modules/slave/manifests/rvm.pp

```diff
@@ -17,6 +17,15 @@
   Class['Rvm::System']
 
   if $facts['rvm_installed'] {
+    if $facts['os']['release']['major'] == '9' {
+      exec { 'install openssl 1.1 through rvm':
+        command  => "rvm pkg install openssl",
+        creates  => "/usr/local/rvm/usr/",
```

Comment:
can this be a more specific path, something that includes openssl? so that if we ever get into the corner needing another `rvm pkg install`, it won't clash?